docs/ml/upload-model.md

```diff
@@ -14,6 +14,7 @@ description: "Upload a Machine Learning model to Viam to use it with the ML Mode
 The ML model service works with models trained inside and outside the Viam app.
 To use a model that you have trained yourself outside the Viam app, [upload it as a new model privately or share it in the Viam registry](#upload-a-new-model-or-new-version).
 If you need to update a previously uploaded model, you can also [upload a new version](#upload-a-new-model-or-new-version).
+To work with the `tflite_cpu` ML model service, an ML model is comprised of a <file>.tflite</file> model file which defines the model, and optionally a <file>.txt</file> labels file which provides the text labels for your model.
```

Comment:
Added first part of sentence because hypothetically other models don't have to use tensorflow right?

Reply:
I believe you are correct! The `tflite_cpu` model indeed requires TF & `.tflite` files specifically, but other models, perhaps delivered as a module, could support their own.

This sentence LGTM!